gui/gfs2img.cpp

```diff
@@ -15,244 +15,249 @@ using namespace std;
 using namespace GMapping;
 using namespace GMapping::GFSReader;
 
-inline double min(double a, double b){
-	return (a<b)?a:b;
+inline double min(double a, double b)
```

Comment:
never reformat code and modify code. Always have two commits to make it easier to review. Too late ...